File: glip-team-messaging/tasks/complete-task/code-samples/completeTask.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$taskId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'status' => 'Incomplete',
    'assignees' => array(
        array(
            'id' => '<ENTER VALUE>'
        )  
    ),
    'completenessPercentage' => 000
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/glip/tasks/{$taskId}/complete", $body);
// PROCESS RESPONSE
?>

File: glip-team-messaging/tasks/complete-task/code-samples/completeTask.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string taskId = "<ENTER VALUE>";

// POST BODY
GlipCompleteTask glipCompleteTask = new GlipCompleteTask {
    status = 'Incomplete',
    assignees = new[] {
        new  {
            id = "<ENTER VALUE>"
        },
    },
    completenessPercentage = 000
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Glip().Tasks(taskId).Complete().Post(glipCompleteTask);
// PROCESS RESPONSE
